async_timeout/__init__.py

```diff
@@ -128,6 +128,13 @@ async def __aexit__(
         self._do_exit(exc_type)
         return None
 
+    def __call__(self, f):
+        async def inner(*args, **kargs):
+            with self:
```

Comment:
```suggestion
            async with self:
```
`with timeout()` is deprecated